pkg/router/f5/plugin.go

```diff
@@ -510,6 +510,28 @@ func (p *F5Plugin) deleteRoute(routename string) error {
 				return err
 			}
 		}
+	} else {
```

Comment:
I think should not use "else" here. That means if passthrough route existed, then no chance to delete reencrypt routes, right?